.github/workflows/build_windows_packages.yml

```diff
@@ -77,13 +74,8 @@ jobs:
           restore-keys: |
             windows-build-packages-v2-
 
-      - name: Install Certs
-        run: python -m pip install certifi
-
       - name: Fetch sources
         run: |
-          # Sets an env variable to ensure urllib uses the certifi CA bundle in fetch_sources.py.
-          export SSL_CERT_FILE=$(python3 -m certifi)
```

Comment:
@amd-chrissosa @Eliasj42 FYI no more certificate setup needed in here now that we use git submodules instead of repo